images/base-ide/README.md

```diff
@@ -0,0 +1,147 @@
+# Theia Base IDE Image
+
+The Base IDE serves as the foundational image for creating language-specific Theia IDE variants. It provides a minimal, lightweight Theia application with essential core functionality, designed to be extended with language-specific tooling and plugins.
+
+## Design Philosophy
+
+- **Minimal**: Includes only universal development tools (Git, Markdown, JSON)
+- **Lightweight**: Excludes language-specific built-ins to reduce image size
+- **Reusable**: Built once, copied into multiple language-specific images
+- **Extensible**: Clean foundation for adding language tooling
+
+## Purpose
+
+This image:
+1. Compiles the Theia IDE application from source
+2. Downloads and configures core VS Code built-in extensions
+3. Provides a standardized base for language-specific images
+4. Eliminates redundant Theia builds across variants
+
+For the complete architecture and usage examples, see the [Images README](../README.md).
+
+## Included VS Code Built-in Extensions
+
+The Base IDE includes a curated subset of VS Code built-in extensions, sourced from Theia's [builtin-extension-pack](https://open-vsx.org/extension/eclipse-theia/builtin-extension-pack). Only universally useful extensions are included to maintain a lightweight base.
+
+### Extension Selection Criteria
+
+Extensions are included if they provide:
+- **Universal functionality**: Useful across all programming languages (Git, Markdown, JSON)
+- **Core IDE features**: Essential for basic development (diff, merge conflict resolution)
+- **Low overhead**: Minimal impact on image size and startup time
+
+### Complete Extension List
+
+The following table shows all available built-in extensions and which are included in this base image:
+
+| Package Name                              | Description | Included |
+|----------------------------------------------|----------|:--------:|
+| eclipse-theia.builtin-extension-pack         | Builtin extension pack associated to a version of vscode | ✔️ |
```

Comment:
This might be confusing for the reader. The extension pack is included, but the extensions below (which are part of the pack) are not. Imho i would only include packs here if they are completely included, extensions included in those packs should not show up in this table. Also a link to the pack (as above) here would be nice.